### ISSUE TITLE
[SDK review] Add ability for an Admin to confirm the meeting scheduled

### ISSUE THREAD

**Issue by maririos** [Azure/azure-sdk-tools#4166]
We are waiting on this issue once we have more information on the flow the admins want for the SDK Review

**Comment by maririos** [comment]
This is no longer needed. We are going with the approach on https://github.com/Azure/azure-sdk-tools/issues/4671
